github4s/jvm/src/test/scala/github4s/unit/DecodersSpec.scala

```diff
@@ -50,4 +51,12 @@ class DecodersSpec extends FlatSpec with Matchers with FakeResponses {
     decode[CombinedStatus](emptyListResponse) should be('left)
   }
 
+  "NonEmptyList Decoder" should "return a valid NonEmptyList for a valid JSON" in {
+    decode[NonEmptyList[Int]]("[1,2,3]") should be(Right(NonEmptyList.of(1, 2, 3)))
+  }
+
+  it should "return an error for an empty list" in {
+    decode[NonEmptyList[Int]](emptyListResponse) should be('left)
+  }
+
```

Comment:
I think it might be worth it to add a test case for a single element because I think that's why this decoder is here for in the first place.

Something like:

```scala
case class Foo(a: Int)
it should "return a valid NonEmtpyList for a valid JSON" in {
  decode[NonEmptyList[Foo]]("""{"a": 1}""") should be(Right(NonEmptyList.of(Foo(1))))
}
```

What do you think?